pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -202,9 +216,9 @@ func (o *capability) handleTransmitMsg(ctx context.Context, resp *response) {
 	}
 
 	select {
+	// This should only happen if the client has closed the upstream context.
```

Comment:
I moved these up because I kept reading this comment and assigning its meaning to `case req.CallbackCh <- resp.CapabilityResponse:`. Are you OK with this @cedric-cordenier?